### PR TITLE
doc(self-managed): drop support of openshift 4.12 and add support of openshift 4.16

### PR DESCRIPTION
**Pull request by leiicamundi**
## Description

OpenShift 4.12 is out of our support matrix since the EOL of the version.
OpenShift 4.16 has been released and tested on our side (https://github.com/camunda/camunda-platform-helm/pull/2131).

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by mesellings** [comment]
@leiicamundi I will review this for you 👍

**Comment by mesellings** [comment]
@leiicamundi I notice you talk about dropping 4.13 support, but have only removed the 4.12 table row - just want to confirm this is correct?

**Review by mesellings — APPROVED** [review]
@leiicamundi I've made review edits directly in the branch:
* Fixed list indentation and numbering
* Some grammar, text changes

Happy to approve 👍

**Review by theburi — APPROVED** [review]

File: docs/self-managed/setup/deploy/openshift/redhat-openshift.md [hunk 1]
```diff
@@ -221,165 +228,166 @@ Additionally, the Zeebe Gateway should be configured to use an encrypted connect
...
+     ingress:
+       grpc:
+         annotations:
+           route.openshift.io/termination: reencrypt
+           route.openshift.io/destination-ca-certificate-secret: camunda-platform-internal-service-certificate
+         className: openshift-default
+         tls:
+           enabled: true
+           secretName: camunda-platform-external-certificate
+   ```
```
Comment (jessesimpson36) [inline docs/self-managed/setup/deploy/openshift/redhat-openshift.md:242]: This PR seems pretty inconsistent. do you prefer indented codeblocks or non-indented codeblocks?
https://github.com/camunda/camunda-docs/pull/4224/files#diff-a2f46593f55132fad493be0c88dcd7ae6aa247e7afaf0bf345d2102fb9161661R76

**Review by jessesimpson36 — APPROVED** [review]
I'm good with the changes.  Personally, I would have preferred if the git-diff wasn't littered with indentation changes but what matters moreso to me is the rendered docs, and those changes are good content.

**Comment by mesellings** [comment]
> I'm good with the changes. Personally, I would have preferred if the git-diff wasn't littered with indentation changes but what matters moreso to me is the rendered docs, and those changes are good content.

Thank you - the indentation is (unfortunately) required by Docusaurus to nest code blocks etc with the relevant list item - I agree it looks messy, but is just how Docusaurus allows you to nest inside lists. 

For example, to have the code block align with the prior list item, you need to add 3 spaces as an indent to trigger this, otherwise it is placed 'outside' the list.

**Pull request merged**